src/styles/base.scss

```diff
@@ -8,6 +8,23 @@
   box-sizing: border-box;
 }
 
+body {
+  background-color: $background;
+}
+
+.visually-hidden:not(:focus):not(:active) {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  margin: -1px;
+  border: 0;
+  padding: 0;
+  white-space: nowrap;
+  clip-path: inset(100%);
+  clip: rect(0 0 0 0);
+  overflow: hidden;
+}
+
```

Comment:
What is this doing? Just curious! :0)

Reply:
Stolen from my bootcamp notes 😆  : 
```There are a few ways of hiding elements, but they all have impacts on accessibility. If we use the property display-none the element is essentially not in the DOM. This means it will not be read by a screen reader. If we use visibility: hidden there will still be a visual footprint, but again, the screen reader will ignore the element. If you want to hide an element but have it read by a screen reader we recommend using some custom CSS often referred to as sr-only or visually-hidden. The best practice for this could evolve over time, but the currently recommended CSS for this class is:

.visually-hidden:not(:focus):not(:active) { 
  position: absolute !important;
  height: 1px; 
  width: 1px;
  overflow: hidden;
  clip: rect(1px 1px 1px 1px); /* IE6, IE7 */
  clip: rect(1px, 1px, 1px, 1px);
  white-space: nowrap; /* added line */
}```

I am using it on one of the labels so that it is still there for accessibility purposes (it will be read by a screen reader) but it is visually hidden to match our mockup design.

Reply:
This is so helpful to know!!